PDSim/scroll/core.py

```diff
@@ -1571,7 +1619,7 @@ def mechanical_losses(self, shell_pressure = 'low:shell'):
             # Get the mean losses over one cycle
             self.losses.bearings  = np.trapz(self.losses.summed[_slice], theta)/theta_range
             
-            print('mechanical losses: ', self.losses.bearings)
+            print('mechanical losses [kW]: ', self.losses.bearings)
```

Comment:
This should go in a separate PR (one topic per PR), but I agree with it entirely!